tests/functionaltests/FunctionalTestHelpers.mm

```diff
@@ -30,11 +30,7 @@
 
 // Launches the functional test app
 bool FunctionalTestSetupUIApplication() {
-    RunSynchronouslyOnMainThread(^{
```

Comment:
I guarantee that it is *not* being called on the UI thread.

UIApplicationInitializeFunctionalTest uses Coredispatcher to call the important parts on the UI thread. 


---
In reply to: [100440302](https://github.com/Microsoft/WinObjC/pull/1937#discussion_r100440302) [](ancestors = 100440302)